### DISCUSSION TITLE
Calculation of hydrodynamic coefficients

### DISCUSSION THREAD

**Discussion by YingjieDuan-hydrod** [capytaine/capytaine#189] [Q&A]
HI!
  Excuse me,in Capytaine,can parameters such as additional mass, additional damping and stiffness of a floating platform be calculated?

**Comment by mancellin** [comment]
Yes, computation of the added mass and radiation damping of a floating body are indeed the main features of Capytaine.
An experimental implementation of the hydrostatic stiffness has also been included in the latest release (1.4).

**Reply by YingjieDuan-hydrod** [reply]
Thanks for your reply, which helps a lot.
